js/main.js

```diff
@@ -153,3 +177,16 @@ document.addEventListener('DOMContentLoaded', function (event) {
     viewSwapDataViews(data.view);
   }
 });
+
+// eventlistener 'click' for document
+document.addEventListener('click', function (event) {
+
+  if (event.target.tagName !== 'A') {
+    return;
+  }
+
+  if (event.target.tagName === 'A' && data.profile.username) {
+    viewSwapDataViews(event.target.getAttribute('data-view'));
+  }
```

Comment:
You need an `else` block here to navigate them to `edit-profile` if they don't have a username (meaning they don't have user profile data)

Reply:
Can I just put “else { return } ?